src/NuGet.Services.KeyVault/KeyVaultReader.cs

```diff
@@ -43,7 +39,8 @@ public async Task<string> GetSecretAsync(string secretName)
 
         public async Task<string> GetSecretAsync(string secretName, ILogger logger)
         {
-            var secret = await _keyVaultClient.Value.GetSecretAsync(_vault, secretName);
+            var response = await _keyVaultClient.Value.GetSecretAsync(secretName);
```

Comment:
The idiomatic way to do this in Azure SDK is with by using the implicit converter.
```suggestion
            KeyVaultSecret response = await _keyVaultClient.Value.GetSecretAsync(secretName);
```
And then eliminating this line: `var secret = response.Value;`